subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy

```diff
@@ -76,11 +76,10 @@ abstract class AbstractSmokeTest extends Specification {
         // https://developer.android.com/studio/releases/build-tools
         static androidTools = "28.0.3"
         // https://developer.android.com/studio/releases/gradle-plugin
-        static androidGradle3x = "3.3.2"
-        static androidGradle = Versions.of("3.2.1", androidGradle3x, "3.4.2", "3.5.0")
+        static androidGradle = Versions.of("3.2.1", "3.3.2", "3.4.2", "3.5.0")
 
         // https://search.maven.org/search?q=g:org.jetbrains.kotlin%20AND%20a:kotlin-project&core=gav
-        static kotlin = Versions.of('1.3.31', '1.3.41', '1.3.50')
+        static kotlin = Versions.of('1.3.20', '1.3.21', '1.3.31', '1.3.41', '1.3.50')
```

Comment:
Do we need to test with both .20 and .21? I think .21 should be enough

Reply:
I'd say if we don't test 1.3.20 then we can't say we support it. It's the lower bound of the supported range.

Reply:
We could say we support 1.3.21 as the lower bound, too. I mean, .21 is a bugfix release, so nobody should be using .20.

Reply:
Alright, I'll ditch 1.3.20 then